src/component.ts

```diff
@@ -1,7 +1,7 @@
 import { nanoid } from "nanoid"
 import { StateTree, ChildMap, EventHandler } from "./purview"
 import { PNodeRegular } from "./types/ws"
-
+import { JSX } from "./purview"
```

Comment:
As you can see, we now have to import `JSX` in any place that the namespace is directly used eg. in references to the type `JSX.Element`.

The branch [this PR](https://github.com/orumhq/orum/pull/3363) is based off of can be merged with the updated version of Purview; it contains all necessary refactors to work together with Purview. Those changes may also be reviewed and merged in isolated fashion if desired (the branch currently also contains React and tRPC changes, which haven't yet been merged to master).

Reply:
To be clear, both React and tRPC can be merged on their own. It's only once any tRPC server route depends on Purview-linked code (for example the User model in`src/models/user`) that the JSX type conflict occurs.